docs/source/essentials/20-subscriptions.mdx

```diff
@@ -1,48 +1,66 @@
 ---
-title: Subscriptions
+title: Subscriptions in Apollo Android
+sidebar_title: Subscriptions
 ---
 
-Queries are useful to fetch data from a server. Mutations are useful to modify data. Subscriptions allow to listen to data changes coming from a server. They work by opening a channel with the server and receiving updates as they are pushed by the server, usually using a websocket.
+Subscriptions are long-lived GraphQL read operations that can update their response over time, enabling clients to receive updates as they occur. They're usually implemented as a connection to a GraphQL server over WebSocket (including in Apollo Android).
 
-Subscriptions are similar to queries in syntax, the only difference being that you use the keyword `subscription` instead of `query` to indicate that the root fields on this query are going to be performing writes to the backend.
+You define a subscription in your app just like you define a query, except you use the `subscription` keyword. Here's an example subscription for getting the latest value of a number whenever that number is incremented:
+
+```graphql
+subscription NumberIncremented {
+  numberIncremented
+}
+```
+
+Unlike with queries and mutations, a subscription operation can include only _one_ field of the `Subscription` type. To subscribe to multiple fields, you create multiple subscription operations.
```

Comment:
TIL! Reading [the discussion in the spec](https://github.com/graphql/graphql-spec/pull/305/files), I'm still not 100% clear what is the root reason for this but good call to include it here 👍